src/Traits/TransactionTrait.php

```diff
@@ -65,6 +70,8 @@ trait TransactionTrait
     /**
      * Transaction description.
      *
+     * @Groups({"create", "update"})
```

Comment:
duplicated with line 77?

Reply:
oops, was re-ordering for alphabetical compliance, fixed